plugin/mouse/mouse.py

```diff
@@ -269,6 +275,10 @@ def show_cursor_helper(show):
 @ctx.action_class("user")
 class UserActions:
     def noise_trigger_pop():
+        # Pop can be configured to wake Talon, in which case we don't want to process it as a click.
+        if not actions.speech.enabled() and setting_mouse_enable_pop_wake.get() == 1:
```

Comment:
The setting is a boolean and you should treat it as such.

Reply:
fixed - also added boolean value to sleep_model.talon

Reply:
Personally I would not use `== True`. It's enough to do `if setting_mouse_enable_pop_wake.get():`

Reply:
removed `== True`